sections/main-cart-items.liquid

```diff
@@ -15,9 +15,16 @@
 
   <div class="cart__warnings">
     <h2 class="cart__empty-text">{{ 'sections.cart.empty' | t }}</h2>
-    <a href="{{ routes.all_products_collection_url }}" class="link">
+    <a href="{{ routes.all_products_collection_url }}" class="button">
       {{ 'general.continue_shopping' | t }}
     </a>
+
+    {%- if shop.customer_accounts_enabled -%}
+      <h3 class="cart__login-title">{{ 'sections.cart.login.title' | t }}</h3>
+      <p class="cart__login-paragraph">
+        {{ 'sections.cart.login.paragraph_html' | t: link: routes.account_login_url }}
+      </p>
+    {%- endif -%}
```

Comment:
Is this only meant to be visible on the empty cart state? 
If it allows you to checkout faster, shouldn't we surface this to customer with items in their cart as well?

Reply:
Yeah, this is empty state only.  It's an interesting point, though.

Reply:
cc. @melissaperreault if you have time to see Martina's question.

Reply:
This is a good question, I'll investigate with the team and come back! 👍